third_party/musl/src/time/gettimeofday.c

```diff
@@ -1,15 +1,24 @@
-#include <time.h>
+#if defined(STARBOARD)
+#include <errno.h>
+#endif
 #include <sys/time.h>
-#if !defined(STARBOARD)
+#include <time.h>
 #include "syscall.h"
-#endif
 
-int gettimeofday(struct timeval *restrict tv, void *restrict tz)
-{
-	struct timespec ts;
-	if (!tv) return 0;
-	clock_gettime(CLOCK_REALTIME, &ts);
-	tv->tv_sec = ts.tv_sec;
-	tv->tv_usec = (int)ts.tv_nsec / 1000;
-	return 0;
+int gettimeofday(struct timeval* restrict tv, void* restrict tz) {
+  struct timespec ts;
```

Comment:
Avoid changing upstream code (keep brace position and tabs, don't change conditional format below)